project.clj

```diff
@@ -5,7 +5,7 @@
     :password :env/clojars_jenkins_password
     :sign-releases false })
 
-(defproject puppetlabs/certificate-authority "0.6.1-SNAPSHOT"
+(defproject puppetlabs/certificate-authority "0.6.2-SNAPSHOT"
```

Comment:
To be more precise, this version number does need to be manually bumped when doing an X or Y release. But the Z number auto-increments.